Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10
+WORKDIR /backup
+
+ENV ACCESS_TOKEN=""
+ENV ORGANIZATION=""
```

Comment:
Think about approach: image should contain only application, not environment. In other words image shouldn't contain any environments.
Your env variables are part of environment.